lib/selection-count-view.coffee

```diff
@@ -1,15 +1,28 @@
+_ = require 'underscore-plus'
+
 class SelectionCountView extends HTMLElement
   initialize: ->
     @classList.add('selection-count', 'inline-block')
 
+    @formatString = atom.config.get('status-bar.selectionCountFormat') ? '(%L, %C)'
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveTextEditor()
 
+    @subscribeToConfig()
     @subscribeToActiveTextEditor()
 
+    toolTipDisposable: null
```

Comment:
This should be above the `initialize: ->` function.

The currently location has the `initialize` method returning a `{toolTipDisposable: null}` object